lib/master.js

```diff
@@ -195,9 +195,35 @@ Master.prototype._startWorkers = function(remainingWorkers) {
                 type: 'config',
                 body: yaml.dump(self.config)
             });
+
+            var workerExit = function(code) {
```

Comment:
@gwicke Change prop wouldn't like that. Its start-up is already flaky as-is, giving up on its sequentiality would only increase it.